### PR TITLE
incubation project to readme.md added

### PR DESCRIPTION
**Pull request by Pratham271**
Added the incubation please check.
Resolving this issue: https://github.com/TBD54566975/developer.tbd.website/issues/731

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -6,12 +6,12 @@ We invite the world to join us in building and adopting open and decentralized t
...
-Components: [tbDEX White paper](https://tbdex.io/whitepaper.pdf), [Decentralized Web Node](https://github.com/TBD54566975/dwn-sdk-js#readme), [Decentralized Identifiers](https://w3c.github.io/did-core/), [Self-Sovereign Identity](https://github.com/TBD54566975/ssi-service#readme), [Self-Sovereign Identity sdk](https://github.com/TBD54566975/ssi-sdk/blob/main/README.md) | Components: [Decentralized Web Node](https://github.com/TBD54566975/dwn-sdk-js#readme), [Decentralized Identifiers](https://w3c.github.io/did-core/), [Self-Sovereign Identity](https://github.com/TBD54566975/ssi-service#readme), [Self-Sovereign Identity sdk](https://github.com/TBD54566975/ssi-sdk/blob/main/README.md), [Self-Sovereign Identity console](https://github.com/TBD54566975/ssi-admin-ui/blob/main/README.md) |
+| [tbDEX](https://developer.tbd.website/projects/tbdex)  | [Web5](https://developer.tbd.website/projects/web5) | [tbLEND](https://discord.gg/bS59ANWc) |
```
Comment (EbonyLouis) [inline profile/README.md]: @Pratham271 Thank you for working on this I think I confused you with my wording on the issue. Apologies right now you have tbLEND under our projects but it needs to go under incubation projects. So it should say 
`We also have two incubation projects with community member project leads` . 

In the picture below I want `tbLEND` to be right next to `Credential Selector`, it should be to the right of it in that chart. Does that make sense? 

Screenshot 2023-10-13 at 1 38 05 PM

Also they now have a GitHub so when you add a link to the name please use this link instead: https://github.com/TBD54566975/incubation-tblend

**Comment by EbonyLouis** [comment]
Also there should only be one file in this PR what is the second file? please remove it

**Comment by Pratham271** [comment]
Deleted the file, please check now.

**Comment by EbonyLouis** [comment]
it's gone thanks! so now it's just addressing the readme changes

**Comment by EbonyLouis** [comment]
@Pratham271 Hey just checking in do you need any help with the changes I requested? we can work on it together

**Comment by Pratham271** [comment]
Can you please tell me the changes that you requested, I think I didn't recieve them

**Comment by Pratham271** [comment]
I wanted to know if there is any update regarding this issue

File: profile/README.md (hunk 1 above)
Comment (EbonyLouis) [reply profile/README.md]: @Pratham271 hey it's what i wrote above, tbLEND needs to be next to Credential Selector in that chart

**Comment by EbonyLouis** [comment]
@Pratham271 in the requested code change, I commented above and tagged you. Where you added the project isn't where I need it to be. If you look at the requested changed above you will see it

**Comment by Pratham271** [comment]
Okay I'll fix that one sorry for the delay

**Comment by Pratham271** [comment]
I have made the changes now please check

File: profile/README.md [hunk 2]
```diff
@@ -6,17 +6,19 @@ We invite the world to join us in building and adopting open and decentralized t
 
  ### Where to start? ⬅️
 
-We currently have two main projects, each project has their components: 
+We currently have three main projects, each project has their components:
```
Comment (EbonyLouis) [inline profile/README.md]: @Pratham271 please put this back to "two main projects"

File: profile/README.md [hunk 3]
```diff
@@ -6,17 +6,19 @@ We invite the world to join us in building and adopting open and decentralized t
...
 
 We also have one incubation project with a community member project lead:
-| [Credential Selector](https://github.com/TBD54566975/incubation-verifiable-credential-selector)  
-| ------------- |
-| Developers embed the VCS into their application to allow users to select and retrieve Verifiable Credentials (VCs) Project Lead: @jacksophtron  
+| [Credential Selector](https://github.com/TBD54566975/incubation-verifiable-credential-selector)  | [tbLEND](https://discord.gg/bS59ANWc) |
```
Comment (EbonyLouis) [inline profile/README.md]: thank you this is exactly where I wanted the new project 🥳

1. On **line 16** please change it from "one incubation project" to "two incubation projects with community member project leads:"
2. on **line 17** you have tbLEND linking to the Discord please link it to the GitHub repo, this is the url "https://github.com/TBD54566975/incubation-tblend#web5service"

File: profile/README.md [hunk 4]
```diff
@@ -6,17 +6,19 @@ We invite the world to join us in building and adopting open and decentralized t
...
+| ------------- | ------------- |
+| Developers embed the VCS into their application to allow users to select and retrieve Verifiable Credentials (VCs) Project Lead: @jacksophtron | Connects the lender and borrower to exchange money securely. Here group of participants that are exchanging data and services are using a free and open source protocol. To know more head over to our [Getting Started](https://photos.onedrive.com/share/177DEE71378DB3AD!9886?cid=177DEE71378DB3AD&resId=177DEE71378DB3AD!9886&authkey=!APppAVFUzYZ2LIY&ithint=video&e=fddgt9)| 
+
+<!-- Components: [Decentralized Web Node](https://github.com/TBD54566975/dwn-sdk-js#readme), [Decentralized Identifiers](https://w3c.github.io/did-core/), [Self-Sovereign Identity](https://github.com/TBD54566975/ssi-service#readme), [Self-Sovereign Identity sdk](https://github.com/TBD54566975/ssi-sdk/blob/main/README.md)| -->
```
Comment (EbonyLouis) [inline profile/README.md]: **Line 21** can you delete this please

File: profile/README.md [hunk 5]
```diff
@@ -6,17 +6,19 @@ We invite the world to join us in building and adopting open and decentralized t
...
 
 We also have one incubation project with a community member project lead:
-| [Credential Selector](https://github.com/TBD54566975/incubation-verifiable-credential-selector)  
-| ------------- |
-| Developers embed the VCS into their application to allow users to select and retrieve Verifiable Credentials (VCs) Project Lead: @jacksophtron  
+| [Credential Selector](https://github.com/TBD54566975/incubation-verifiable-credential-selector)  | [tbLEND](https://discord.gg/bS59ANWc) |
+| ------------- | ------------- |
+| Developers embed the VCS into their application to allow users to select and retrieve Verifiable Credentials (VCs) Project Lead: @jacksophtron | Connects the lender and borrower to exchange money securely. Here group of participants that are exchanging data and services are using a free and open source protocol. To know more head over to our [Getting Started](https://photos.onedrive.com/share/177DEE71378DB3AD!9886?cid=177DEE71378DB3AD&resId=177DEE71378DB3AD!9886&authkey=!APppAVFUzYZ2LIY&ithint=video&e=fddgt9)|
```
Comment (EbonyLouis) [inline profile/README.md]: Please change tbLEND's description to this: 
"A protocol that connects the lender and borrower to exchange money securely. To learn more about their project you can get started by watching this short [project overview](https://photos.onedrive.com/share/177DEE71378DB3AD!9886?cid=177DEE71378DB3AD&resId=177DEE71378DB3AD!9886&authkey=!APppAVFUzYZ2LIY&ithint=video&e=fddgt9). Project Lead: @ianpatton"

**Comment by EbonyLouis** [comment]
@Pratham271 thank you for the changes it's almost done 🥳. I added just a few things for you to change you can find them in the requested changes in the code review.

**Review by EbonyLouis — CHANGES_REQUESTED** [review]
@Pratham271 last thing, Remove this file from this PR  `.DS_Store` pls

**Comment by Pratham271** [comment]
Hi @EbonyLouis please check now

File: .DS_Store
Comment (EbonyLouis) [inline .DS_Store:1]: @Pratham271 the file is still here

**Comment by EbonyLouis** [comment]
> Hi @EbonyLouis please check now
I left a comment, the file is still apart of this PR @Pratham271

**Comment by Pratham271** [comment]
Now the file is not included

**Comment by EbonyLouis** [comment]
Thank you soo much ! great work! @Pratham271

**Review by EbonyLouis — APPROVED** [review]

**Pull request merged**